lib/puppet/parser/functions/lookup.rb

```diff
@@ -1,52 +1,57 @@
 Puppet::Parser::Functions.newfunction(:lookup, :type => :rvalue, :arity => -2, :doc => <<-'ENDHEREDOC') do |args|
-Looks up data defined using Puppet Bindings and Hiera.
-The function is callable with one to three arguments and optionally with a code block to further process the result.
+Looks up data defined using Hiera and Data Providers
```

Comment:
"Data Bindings" instead of "Hiera" (even if it is the only implementation of Data Bindings...